nepal_compliance/__init__.py

```
@@ -1 +1,16 @@
 __version__ = "0.2.0"
+
+# Monkey patch Frappe's in_words function to use Nepali number-to-words conversion
+import frappe.utils
+import frappe.utils.data
+
+def _apply_patches():
```

Ruff (0.15.0)

[warning] Missing return type annotation for private function `_apply_patches`

Add return type annotation: `None`

(ANN202)